fyle_rest_auth/helpers.py

```diff
@@ -78,7 +78,10 @@ def validate_code_and_login(request):
              and settings.FYLE_REST_AUTH_SETTINGS['async_update_user']:
             async_task(
                 'apps.workspaces.tasks.async_update_fyle_credentials',
-                employee_info['data']['org']['id'], tokens['refresh_token']
+                employee_info['data']['org']['id'], tokens['refresh_token'],
+                q_options={
```

Comment:
this can go to the default cluster, lets not put anything backward incompatible here